mathics/doc/doc.py

```diff
@@ -714,39 +714,40 @@ def load_pymathics_doc(self):
             return
         from mathics.settings import default_pymathics_modules
         pymathicspart = None
-        # Look the "Pymathics Modules" part, and if it does not exist, create it.  
+        # Look the "Pymathics Modules" part, and if it does not exist, create it.
         for part in self.parts:
             if part.title == "Pymathics Modules":
                 pymathicspart = part
         if pymathicspart is None:
             pymathicspart = DocPart(self, "Pymathics Modules", is_reference=True)
             self.parts.append(pymathicspart)
 
-        # For each module, create the documentation object and load the chapters in the pymathics part.    
+        # For each module, create the documentation object and load the chapters in the pymathics part.
         for pymmodule in default_pymathics_modules:
             pymathicsdoc = PyMathicsDocumentation(pymmodule)
             for part in pymathicsdoc.parts:
-                if part.title == "Pymathics Modules":
-                    for ch in part.chapters:
-                        ch.part = pymathicspart
-                        pymathicspart.chapters_by_slug[ch.slug] = ch
-                        pymathicspart.chapters.append(ch)
+                for ch in part.chapters:
+                    ch.title = pymmodule.name + " " + part.title + " " + ch.title
```

Comment:
This could be improved with and `f`-string: `ch.title = f"{pymmodule.name} {part.title} {ch.title}"`.